backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -730,6 +738,8 @@ class SchedulingService implements ISchedulingService {
         } for ${startDayString} at ${startTimeString}${
           isRecurringDonation ? " and all following donations " : ""
         } has been cancelled.`;
+        // check if start time string != volnteer start time string stuff
```

Comment:
will delete this comment once i get clarification and make appropriate changes if necessary

basically, does this show schedule time or volunteer time?

Reply:
removed this and displayed volunteer's start time in the email